packages/ua-devtools-evm/src/ownable/mixin.ts

```diff
@@ -0,0 +1,31 @@
+import { OmniAddress, OmniTransaction, areBytes32Equal, mapError } from '@layerzerolabs/devtools'
+import { OmniSDK } from '@layerzerolabs/devtools-evm'
+import type { IOwnable } from '@layerzerolabs/ua-devtools'
+
+export const OwnableMixin: IOwnable = {
+    async getOwner(this: OmniSDK): Promise<OmniAddress> {
+        this.logger.debug(`Getting owner`)
+
+        const owner = await mapError(
+            async () => this.contract.contract.owner(),
+            (error) => new Error(`Failed to get owner for OApp ${this.label}: ${error}`)
+        )
+
+        return this.logger.debug(`Got owner: ${owner}`), owner
```

Comment:
For my own education, does this syntax just mean "return owner after I log this debug thingy"?

Reply:
Yeah exactly, comma just means execute these expressions in sequence (and return the last one in this particular case). Works in many places, e.g.

```typescript
# Loops
for (let i = 0, j = 0; i++, j++; i < 10) {}

# Variables (much discouraged but very prevalent 20 years ago)
const I = 0, j = 0, k = 0;

# The return example above
return doSomething(), doSomethingElse(), getWhatNeedsToBeGot()
```